compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STGroupByClauseNode.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
This is a known issue in treeGen. We have to manually update the copyright header in all the auto-generated files